Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/search/SearchViewModel.kt

```diff
@@ -0,0 +1,52 @@
+package com.mentalmachines.droidcon_boston.views.search
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.google.firebase.database.DataSnapshot
+import com.google.firebase.database.DatabaseError
+import com.google.firebase.database.ValueEventListener
+import com.mentalmachines.droidcon_boston.data.FirebaseDatabase
+import com.mentalmachines.droidcon_boston.data.Schedule
+import com.mentalmachines.droidcon_boston.firebase.FirebaseHelper
+import timber.log.Timber
+
+class SearchViewModel : ViewModel() {
+    private val firebaseHelper = FirebaseHelper.instance
+    private val conferenceYear = "2018"
```

Comment:
I know this isn't great, but it's temporary. Once we merge this, I'd like to do another small PR where I consolidate everywhere that we put the conference dates. We do it in a few spots, and I think it would be good to put them in like a build config variable or something like that. but I wanna keep it separate.

Reply:
why do we even care about the year? We should just accept what the script is returning, unless the script is adding all years to the database?

Reply:
For sure. This would of course require another change to match up with 2019 once @kenyee wraps up the import script. I'm okay with this, maybe add a TODO here just to ensure this isn't missed!

Reply:
Add a TODO here to ensure this is ready for the 2019 data.